package.json

```diff
@@ -22,32 +22,31 @@
         "preversion": "npm run lint",
         "NOTYET-version": "npm run format && git add -A src",
         "NOTYET-postversion": "git push && git push --tags",
-        "prepare": "npm run build"
+        "prepare": "husky install"
```

Comment:
Did you mean to switch the `prepare` script from `npm run build` to `husky install`? They seem to me like two separate commands.
I notice in both techlag and the docs I looked at, that they recommend calling the script for husky install `"postinstall": "husky install"`

Reply:
code was updated to :  
"prepare": "npm run build",
"postinstall": "husky install"